datalad/support/network.py

```diff
@@ -900,22 +901,24 @@ def is_ssh(ri):
         or (isinstance(_ri, URL) and _ri.scheme == 'ssh')
 
 
-def get_local_file_url(fname):
+def get_local_file_url(path):
```

Comment:
Grepping around the extensions, I don't see any calls to `get_local_file_url` that use `fname` as a keyword argument, so changing the parameter name should be fine.  But at the same time, any subjective benefit of the name change doesn't seem to be worth making a breaking change.

If we do stick with name change, the docstring needs to be updated.

Reply:
You are right, I will revert it.

Reply:
Done